macos/FluentUI/Button/Button.swift

```diff
@@ -366,20 +366,34 @@ open class Button: NSButton {
 // MARK: - ButtonCell
 
 class ButtonCell: NSButtonCell {
-	fileprivate var verticalPadding: CGFloat = 0
-	fileprivate var horizontalPadding: CGFloat = 0
-	fileprivate var titleVerticalPositionAdjustment: CGFloat = 0
-	fileprivate var titleToImageSpacing: CGFloat = 0
-	fileprivate var titleToImageVerticalSpacingAdjustment: CGFloat = 0
+	var verticalPadding: CGFloat = 0
```

Comment:
Should be private for these rather than inheriting value from ButtonCell (internal?)

Reply:
`Button` needs to set these values on its cell, so `private` is not an option.  Rafael said `fileprivate` is bad.  `internal` is the next least permissive option.

Reply:
I don't know if it's bad, I was just saying we should be consistent with our documentation if we decided not to like `fileprivate` anymore. Ironically, the comment in my PR to update documentation was "So then, what is a good example of `fileprivate`? Aaaannnnnnnnddd I think your scenario (A Button and a ButtonCell in the same file) is a perfect example 🙃

Reply:
@rdeassis what is the specific rule that prompted the original `fileprivate` commant?  Is it to prefer `private` over `fileprivate` in the outermost scope?  Is `fileprivate` fine in narrower scopes, such as the uses I had here?

Reply:
That was my understanding - avoid using fileprivate where private is identical (outermost scope). I agree that this is a good case for fileprivate

Reply:
@chripear, below is the warning with the reference of samples of code that trigger it :

[Private over fileprivate Violation: Prefer `private` over `fileprivate` declarations. (private_over_fileprivate)](https://realm.github.io/SwiftLint/private_over_fileprivate.html)

https://realm.github.io/SwiftLint/private_over_fileprivate.html